addons/mail/static/src/models/messaging.js

```diff
@@ -238,18 +234,15 @@ registerModel({
             for (const guest of this.models['Guest'].all()) {
                 guestIds.push(guest.id);
             }
-            if (partnerIds.length !== 0 || guestIds.length !== 0) {
-                const dataList = await this.messaging.rpc({
-                    route: '/bus/im_status',
-                    params: {
-                        partner_ids: partnerIds,
-                        guest_ids: guestIds,
-                    },
-                }, { shadow: true });
-                this.models['Partner'].insert(dataList.partners);
-                if (dataList.guests) {
-                    this.models['Guest'].insert(dataList.guests);
-                }
+            if (partnerIds.length) {
```

Comment:
This is conflicting with line 230, which would also prevent from reaching the guest case.